### ISSUE TITLE
Questions about retrain the model.

### ISSUE THREAD

**Issue by Jackie666666** [umitkacar/Ear-segmentation-ai#2]
Hi, Mr. Umitkacar, 
I want use your pre-train model to do ear segmentation in my images dataset. 
In my photo, there is no head or face, so your model could not segment the ear,
so I want to fine-tune your model use my images dataset,
but the loss could not converge, and the result is terrible.
I would like to ask for the source training code of your model.
Thank you very much.

**Comment by umitkacar** [comment]
Any image should have a head, face and background. Because there are images in this format in the dataset. Your scenario is very different. Can you share an example image?

**Comment by Jackie666666** [comment]
> Any image should have a head, face and background. Because there are images in this format in the dataset. Your scenario is very different. Can you share an example image?
Thank you. This is the image of my dataset.
![0001](https://user-images.githubusercontent.com/45167969/180251609-d711a4cd-be6e-4003-a100-23ce37310774.png)
There is no background in the image, and according your explain, should I train a new model from the scratch?
and what is the loss function & optim function that you utilized in your training  procedure?

**Comment by umitkacar** [comment]
You can use this repo:
[https://github.com/qubvel/segmentation_models.pytorch](url)

**Comment by Jackie666666** [comment]
> You can use this repo: https://github.com/qubvel/segmentation_models.pytorch

Ok, I am using the repo right now, and the result is better,
Thank you very much.
